### PR TITLE
Fixed DragonFly BSD build (patches from dports).

### PR DESCRIPTION
**Pull request by realzhtw**
These are two patches from the oficial DragonFly BSD dports collection.

### CODE REVIEW

File: src/unix/freebsd.c [hunk 1]
```diff
@@ -58,7 +58,8 @@ void uv__platform_loop_delete(uv_loop_t* loop) {
 
 
 #ifdef __DragonFly__
-int uv_exepath(char* buffer, size_t* size) {
+/* deprecated long ago, use KERN_PROC_PATHNAME sysctl */
+int uv_exepath_procfs(char* buffer, size_t* size) {
```
Comment (vtjnash) [inline src/unix/freebsd.c:62]: Since this is now unused from anywhere, seems like it should be deleted?

Comment (realzhtw) [reply src/unix/freebsd.c:62]: Well, I am not the original author of the patch. Why it wasn't simply deleted - I can only guess. To me this indeed should be deleted. I will create a separate PR then.

**Review by cjihrig — APPROVED** [review]

**Comment by cjihrig** [comment]
CI: https://ci.nodejs.org/view/libuv/job/libuv-test-commit/2037/

**Comment by cjihrig** [comment]
Squashed and landed in https://github.com/libuv/libuv/commit/113642075853a7ebdd9867b0e097fce4bc7ff873. Thanks!

**Pull request closed without merging**